### ISSUE TITLE
realloc() invalid next size - gff2bed - v2.4.21 - Linux

### ISSUE THREAD

**Issue by ObjectiveTruth** [bedops/bedops#167]
Following the Linux install guide:

http://bedops.readthedocs.io/en/latest/content/installation.html#linux

I installed the 64-bit binaries from here (v2.4.21)

https://github.com/bedops/bedops/releases

And when running on the sample dataset gff from the website:

http://bedops.readthedocs.io/en/latest/content/reference/file-management/conversion/gff2bed.html#downloads

I get this error:

```
*** Error in `convert2bed': realloc(): invalid next size: 0x0000000000973060 ***
/home/ec2-user/bin/gff2bed: line 132: 32082 Aborted                 ${cmd} ${options} - 0<&0
```

I tried it using `Amazon Linux AMI` and `Fedora25` and had the same issue

### Workaround:

Installed the `v2.4.20` binaries and worked as expected

**Comment by alexpreynolds** [comment]
I did not reproduce this on OS X but it does occur on Ubuntu 14. I'm investigating the cause.

**Comment by alexpreynolds** [comment]
Thanks for the catch. I found a heap corruption issue with the GFF attributes field that causes subsequent reallocs of that field to fail. I'll push the fix out shortly in v2.4.22.

**Comment by alexpreynolds** [comment]
Fixed in commit 3c8df89
